source/features/tags-on-commits-list.tsx

```diff
@@ -49,43 +50,12 @@ function isTagTarget(target: CommonTarget): target is TagTarget {
 }
 
 async function getTags(lastCommit: string, after?: string): Promise<CommitTags> {
-	const {repository} = await api.v4(`
-		repository() {
-			refs(
-				first: 100,
-				refPrefix: "refs/tags/",
-				orderBy: {
-					field: TAG_COMMIT_DATE,
-					direction: DESC
-				}
-				${after ? `, after: "${after}"` : ''}
-			) {
-				pageInfo {
-					hasNextPage
-					endCursor
-				}
-				nodes {
-					name
-					target {
-						commitResourcePath
-						... on Tag {
-							tagger {
-								date
-							}
-						}
-						... on Commit {
-							committedDate
-						}
-					}
-				}
-			}
-			object(expression: "${lastCommit}") {
-				... on Commit {
-					committedDate
-				}
-			}
-		}
-		`);
+	const {repository} = await api.v4(GetTagsOnCommit, {
+		variables: {
+			after: after ?? null,
```

Comment:
How is this passing lint? 😩 Unreal


Anyway, it's the result of accepting a Json type

Reply:
Why shouldn't it be passing lint? :thinking:

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-null.md

Reply:
xo disabled `no-null`. https://github.com/xojs/xo/blob/main/config/plugins.cjs#L163

Reply:
Oh I thought I had it enabled here. I've been switching between projects where this was enforced